### PR TITLE
fix(docs): bad formatting in the Ingress API reference

### PR DESCRIPTION
**Pull request by elizabeth-dev**
IngressSpec rules.http.paths.pathType property had a bad formatted list on its description

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1.md [hunk 1]
```diff
@@ -204,13 +204,18 @@ IngressSpec 描述用户希望存在的 Ingress。
       - **rules.http.paths.pathType** (string)，必需
 
         <!--
-        pathType determines the interpretation of the path matching. PathType can be one of the following values: * Exact: Matches the URL path exactly. * Prefix: Matches based on a URL path prefix split by '/'. Matching is
+        pathType determines the interpretation of the path matching. PathType can be one of the following values:
```
Comment (elizabeth-dev) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1.md:207]: I'm not sure if I should have also edited this in the zh-cn locale, or if it's done automatically. Let me know if I should leave that as is.

(Also sorry for the whitespace trimming changes, IDE did that, hope it's okay)

Comment (dipesh-rawat) [reply content/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1.md:207]: The documentation follows distinct processes for each localization, and we typically do not accept PR that impact multiple languages simultaneously. The recommendation is splitting PR into multiple pull requests, with each one concentrating on a single language.

Additionally, please note that this page is auto-generated, so the fix need to be implemented elsewhere. You may find the following links helpful for further guidance:

- For generating reference docs and contributing upstream: [kubernetes.io/docs/contribute/generate-ref-docs/contribute-upstream](https://kubernetes.io/docs/contribute/generate-ref-docs/contribute-upstream/)
- Building the API reference pages: [github.com/kubernetes/website/blob/main/README.md#building-the-api-reference-pages](https://github.com/kubernetes/website/blob/main/README.md#building-the-api-reference-pages)

Comment (howieyuen) [reply content/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1.md:207]: > I'm not sure if I shouldn't have edited this in the zh-cn locale, or if it's done automatically. Let me know if I should leave that as is.
> 
> (Also sorry for the whitespace trimming changes, IDE did that, hope it's okay)

You don't need change zh-cn locale, we sig-doc-zh team will do this sync job.

Comment (elizabeth-dev) [reply content/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1.md]: @dipesh-rawat @howieyuen fixed!

**Comment by elizabeth-dev** [comment]
@kundan2707 `content/en/docs/reference/kubernetes-api/service-resources/ingress-v1.md` is auto-generated?

**Comment by sftim** [comment]
> @kundan2707 `content/en/docs/reference/kubernetes-api/service-resources/ingress-v1.md` is auto-generated?

Yes, that's right. See https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/ (search for “auto-generated”)

**Comment by kundan2707** [comment]
@elizabeth-dev Please change accordingly to fix this issue.

**Comment by divya-mohan0209** [comment]
@elizabeth-dev : Making changes to the reference docs entails a wholly different process. Please check [this section](https://kubernetes.io/docs/contribute/generate-ref-docs/) in our contributor guide to learn more.
/close

**Pull request closed without merging**